### ISSUE TITLE
Remove com.android.vending.BILLING from Manifest

### ISSUE THREAD

**Issue by s8mathur** [RevenueCat/purchases-android#210]
Since integrating with the RevenuCat Android library, Play Store Pre-Launch test report shows the following APK Security warning about the use of com.android.vending.BILLING . Prior to RC, our app was using Billing Client library directly and not triggering this warning.

Request that the RevenueCat manifest remove it if no longer necessary for modern API levels and Play Store Billing Client versions

> Anomalous Permission Usage
> Detected in APK 141, 140, 139, 138, 137, 136, 135, 134, 133, 132 ...
> Your app is requesting the following permission which is used by less than 10% of functionally similar apps: com.android.vending.BILLING
> 
> Users prefer apps that request fewer permissions and requesting unnecessary permissions can affect your app's visibility on the Play Store. If these permissions aren't necessary, you may be able to use alternative methods in your app and request fewer permissions. If they are, we recommend providing an explanation to users of why you need the permissions. Learn more
> 
> Note: This guidance is based on a comparison with functionally similar apps, which change over time as new apps get published and existing apps change behavior. Therefore the warning may change even if you don't change your permission usage.

**Comment by rkotzy** [comment]
I believe this is a required permission to make in-app purchases through Google Play Billing: https://developer.android.com/google/play/billing/integrate#dependency

**Comment by s8mathur** [comment]
Hey Ryan,
Is the RC library Manifest listing it explicitly? 
My understanding is that as app developers and library providers, we are no longer expected to add it to our manifests. Please see the the v1.0 release notes for Billing library: https://developer.android.com/google/play/billing/release-notes#release-1_0

> Embedded billing permission inside library’s manifest. It's not necessary to add the com.android.vending.BILLING permission inside Android manifest anymore.

Please also see: https://developer.android.com/google/play/billing/migrate#remove

> Remove the com.android.vending.BILLING permission
> The Google Play Billing Library embeds the com.android.vending.BILLING permission inside its manifest. It is no longer necessary to explicitly add this permission inside your app's manifest.

**Comment by rkotzy** [comment]
Thanks for the additional details @s8mathur. Going to tag @vegaro to investigate since they'll know more than me about this.

**Comment by vegaro** [comment]
We can remove it from our `AndroidManifest.xml`, but the BillingClient has it in its own `AndroidManifest.xml`:

```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.android.billingclient"
    android:versionName="3.0.1" >

    <uses-sdk
        android:minSdkVersion="14"
        android:targetSdkVersion="22" />

    <uses-permission android:name="com.android.vending.BILLING" />

    <application>
        <meta-data
            android:name="com.google.android.play.billingclient.version"
            android:value="3.0.1" />

        <activity
            android:name="com.android.billingclient.api.ProxyBillingActivity"
            android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
            android:exported="false"
            android:theme="@android:style/Theme.Translucent.NoTitleBar" />
    </application>

</manifest>
```

When merging manifests, your app will end up with the BILLING permission anyhow even if we remove it from our `AndroidManifest.xml` since the BillingClient is adding it. I am not sure it will remove the warning you are seeing but I will remove it just in case it helps

**Comment by vegaro** [comment]
Released in version 4.0.0
